src/develop/payment.md

```diff
@@ -0,0 +1,334 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```shell
+npm install graphql
+npm install -D typescript
+npm install -D @graphql-codegen/cli
+npm install -D @graphql-codegen/typescript
+```
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```shell
+yarn add graphql
+yarn add -D typescript
+yarn add -D @graphql-codegen/cli
+yarn add -D @graphql-codegen/typescript
+```
+  </TabItem>
+</Tabs>
+
+Next, create a file named `codegen.ts` in your project's root folder containing the following (adjust as needed):
+
+```typescript
+import type {CodegenConfig} from '@graphql-codegen/cli'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+const schemaUrl = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: [
+    {
+      [schemaUrl]: {
+        headers: {
+          Authorization: `Basic ${PROJECT_SECRET_BASE64}`,
+        },
+      },
+    },
+  ],
+  documents: ['src/**/*.ts'],
+  ignoreNoDocuments: true, // for better experience with the watcher
+  generates: {
+    'src/generated/graphql.ts': {
+      plugins: ['typescript'],
+    },
+  },
+  watch: true,
+}
+
+export default config
+```
+
+Finally, follow [these instructions](https://the-guild.dev/graphql/codegen/docs/getting-started/development-workflow) to add GraphQL code generation to your development workflow.
+
+## API initialization
+
+To start making queries to the GraphQL API, you'll need to configure your client. Following is an example on how you can do that using either the Everscale SDK or Axios.
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {TonClient} from '@eversdk/core'
+import {libNode} from '@eversdk/lib-node'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+TonClient.useBinaryLibrary(libNode)
+
+const client = new TonClient({
+  network: {
+    endpoints: [`https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`],
+    access_key: PROJECT_SECRET_BASE64,
+  },
+})
+```
+  </TabItem>
+  <TabItem value="axios" label="Axios">
+
+```typescript
+import axios from 'axios'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+axios.defaults.baseURL = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+axios.defaults.headers.post['Content-Type'] = 'application/json'
+if (PROJECT_SECRET_BASE64) {
+  axios.defaults.auth = {
+    username: '',
+    password: PROJECT_SECRET_BASE64,
+  }
+}
+```
+  </TabItem>
+</Tabs>
+
+## Code examples
+
+Following examples demonstrate how you can use the types generated from the GraphQL schema in your project.
+
+### Get account balance
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {BlockchainQuery} from './generated/graphql'
+
```

Comment:
Please add the Ever SDK import so that the example is reproducible

Reply:
These imports are included in the previous section (API initialization), I figured it makes more sense that way. Otherwise we would have to remove that section altogether and duplicate the same code in all examples. If there are more examples added later, it would become quite redundant, no?

Reply:
Mayble I'll just add a comment to add the import at the top of the file with the other imports?

Reply:
> These imports are included in the previous section (API initialization), I figured it makes more sense that way. Otherwise we would have to remove that section altogether and duplicate the same code in all examples. If there are more examples added later, it would become quite redundant, no?

hmm, I agree, yes

Reply:
> Mayble I'll just add a comment to add the import at the top of the file with the other imports?

Sounds like a good idea